src/duplo_resource/configmap.py

```diff
@@ -14,8 +14,12 @@ def __init__(self, duplo: DuploClient):
   def list(self):
     """Retrieve a list of all configmaps in a tenant."""
     tenant_id = self.tenant["TenantId"]
+    tenant_name = self.tenant["AccountName"]
     response = self.duplo.get(f"v3/subscriptions/{tenant_id}/k8s/configmap")
-    return response.json()
+    if (data := response.json()):
```

Comment:
We really can just `return response.json()` because the validate response will catch the 404.